Source/FieldSolver/FiniteDifferenceSolver/Make.package

```diff
@@ -8,6 +8,8 @@ CEXE_sources += MacroscopicEvolveE.cpp
 #ifdef WARPX_MAG_LLG
 CEXE_sources += MacroscopicEvolveM.cpp
 CEXE_sources += MacroscopicEvolveM_2nd.cpp
```

Comment:
if we decide that we dont need MacroscopicEvolveM.cpp, we can remove these from compilation.